backend/src/controllers/SpotController.js

```diff
@@ -20,15 +20,17 @@ module.exports = {
     if (!user) {
       return res.status(400).json({ error: 'User does not exists' });
     }
+    else{
+      const spot = await Spot.create({
+        user: user_id,
+        thumbnail: filename,
+        company,
+        techs: techs.split(',').map(tech => tech.trim()),
+        price
+      })
+    }
```

Comment:
You don't need the `else` here. If the user doesn't exists the `return` will be called so the code will stop there.

Reply:
theoretically you're right, but for some reason when i am testing the app the code still executes after the `return` and creates the spot when the user doesn't exist, the only workaround that i've found is to put the `else` statement